c_emulator/riscv_platform.c

```diff
@@ -21,6 +21,15 @@ bool sys_enable_fdext(unit u)
 bool sys_enable_zfinx(unit u)
 { return rv_enable_zfinx; }
 
+bool sys_enable_smepmp(unit u)
+{ return ( (rv_enable_Smepmp == 0) ? false : true); }
```

Comment:
(a) the formatting is atrocious (you don't need any of those parentheses, nobody writing C does that, it's way too cluttered, and even if you did you wouldn't put a space between the two `(`s)

(b) `expr ? false : true` and `expr ? true : false` are code smells, one should use `expr` and `!expr` accordingly

(c) ... which in this case would then be != 0 rather than `!(rv_enable_smepmp == 0)`